solutions/swb-reference/README.md

```diff
@@ -68,11 +68,232 @@ This step is necessary to setup Service Catalog portfolio and products
+                    "id": "########-####-####-####-############",
+                    "pk": "ETC",
+                    "productId": "<product id>",
+                    "type": "sagemakerNotebook"
+                }
+            ]
+        }
+        ```
+        If there are no environment type configs displayed please follow instructions [here](../swb-reference/SETUP_v2p1.md###setup-project-configurations,-environmentType,-and-environmenttypeconfig) to create a new environment type config.
+
+    3. Choose the Environment Type Config that integration test will use as default when creating any Environment and copy the `id` value from the request.
+
+    4. In `./integration-tests/config` directory assign value copied to `envTypeConfigId` property in `<STAGE>.yaml` file 
+
+
+4. For `projectId` open Postman Collection and select `List projects` inside `projects` folder.
+
+    1. Excecute `List projects` request, you should get a json response with the next information
+        ```
+        {
+            "data": [
+                {
+                    "subnetId": "subnet-<subnet id>",
+                    "accountId": "<account id>",
+                    "awsAccountId": "<Hosting account id>",
+                    "environmentInstanceFiles": "s3://swb-dev-va-<bucket id>/environment-files",
+                    "createdAt": "2022-01-28T22:42:20.296Z",
+                    "updatedBy": "abc",
+                    "vpcId": "vpc<vpc id>",
+                    "envMgmtRoleArn": "arn:aws:iam::<hosting account id>:role/swb-dev-va-env-mgmt",
+                    "createdBy": "abc",
+                    "name": "Project 1",
+                    "encryptionKeyArn": "arn:aws:kms:us-east-1:<hosting account id>:key/########-####-####-####-############",
+                    "resourceType": "project",
+                    "externalId": "workbench",
+                    "updatedAt": "2022-02-02T21:07:30.237Z",
+                    "projectAdmins": [],
+                    "indexId": "index-123",
+                    "sk": "PROJ#<proj id>",
+                    "owner": "abc",
+                    "description": "Example project 1",
+                    "id": "########-####-####-####-############",
+                    "pk": "########-####-####-####-############",
+                    "hostingAccountHandlerRoleArn": "arn:aws:iam::<hosting account id>:role/swb-dev-va-hosting-account-role"
+                }
+            ]
+        }
+        ```
+
+    2. Choose the Project that integration test will use as default when creating any Environment and copy the `id` value from the request.
+
+    3. In `./integration-tests/config` directory assign value copied to `projectId` property in `<STAGE>.yaml` file 
+
+
+5. For `terminatedEnvId` we need the id of an environment that has been terminated, Postman collection request `List Environments` does not show terminated environments ,so we need to save the id of a stopped environment before we terminate it.
```

Comment:
Why don't we have the integ test launch, stop, and terminate an environment so it knows what the id of the environment to use is?

Reply:
we do for multi step, there is one isolated test that requires a terminated one to run, IMO we should delete the isolated test for the terminated env, it only validates we cannot terminate an already terminated environment, creating, stopping and terminating an environment for this test means we have to add another  ~11 minutes to the test

Reply:
In the multi step, do we test that we cannot terminate an already terminated environment?

Reply:
we dont, but it would be easier to implement there instead

Reply:
Yeah, let's do that. Should we create another ticket for that improvement and merge in before this one?

Reply:
This PR was created for a request of a user to clarify what was needed in the yaml file for integration test to work
So I will try to push this PR and I already created a [ticket](https://sim.amazon.com/issues/CHAM-178) to improve this and update the documentation